html/php-pre-challenge1/index.php

```diff
@@ -1,5 +1,14 @@
 <?php
 for ($i = 1; $i <= 100; $i++) {
     //ここからコードを書く
-    echo $i;
+    if ($i%3 === 0 && $i%5 === 0) {
```

Comment:
間違いではないですが、ちなみにこうするとちょっと高速化します！せっかくなので修正してみましょう！
```suggestion
    if ($i % 15 === 0) {
```